### ISSUE TITLE
Rough Agenda

### ISSUE THREAD

**Issue by bdougie** [bdougie/codeNaija#12]
Codenaija agenda
Agenda Day 1:

- 8:30am: Breakfast and registration (‘Toya + Jared + Stan)
- 9:15am: Opening ceremony and remarks
    - 5min GitHub welcome 
    - 15min Andela
    - 5min each Other partner intros (Africa Azure?, Flutterwave, M12)
- 8:45am Ice Break (Jared)
- 9:45am: IceBreaker
- 10am Keynote speaker on “The Digital Divide”
- 10:40am: Learn about how to contribute to Open-source from a prominent maintainers on GitHub.
- 1045am: 15min break
- 11am Open Source Maintainer show and tell (maybe move this to the written form and 
- 12:00pm Lunch
- 1:00pm GitHub Panel
- 1:00pm: Let the Hacking begin
    - During the hacking, there will be pomodoro (25min) checkins via slack and in-person to unblock teams from mentors
- 2:25pm: GitHub Tips/Tricks (Technical tips and tricks)
- 3:25pm: Continue Hacking
- 4:25pm: Group Technical Workshop 1 (workshop or show/tell)
- 5:00pm: Hacking ends and dinner begins
- 6:00pm: Afterparty/Mixer
- 8:00pm: Experience ends

Agenda Day 2:

- 8:30am: Breakfast and continued hacking
- 9:15am: Opening ceremony and remarks
- Keynote from Local Open Source for Good or Technology Representative
- 10am: Hacking continues
- 12:00pm Lunch
- 1:00pm: Hacking
- 3:25pm: Hacking Ends
- 3:30pm: Group Career Development Workshop (judge deliberation)
- 430pm: 
- 6:00pm: Dinner begins
- 6:30pm: Afterparty
- 10:00pm: Experience ends

Rubric

Teams must build a project that shares knowledge across the digital divide within Africa.

**Comment by bdougie** [comment]
closing in favor of #43
